README.md

```diff
@@ -3,3 +3,20 @@ libadiff - Audio diffing library
 
 [![Build Status](https://travis-ci.org/concert/libadiff.svg?branch=master)](
     https://travis-ci.org/concert/libadiff)
+
+## Build:
+If you're on a Debian-based Linux system, prerequisites are:
+```
+apt-get install libglib2.0-dev libsndfile-dev
+pip install meson ninja  # Python3
```

Comment:
This differs from meson's install guide (which maybe we should just link): http://mesonbuild.com/Quick-guide.html

Reply:
I thought you didn't want to be all `apt`y? ;-)
I think a link to the meson build system is a good idea. Perhaps somewhere we can just mention that we use the `[meson](http://mesonbuild.com/Quick-guide.html)` build system, and just use that as a nice, clean implicit link to setting up.
However, I think a set of instructions in one place for getting this to work is better for usability, and I'm not sure that suggesting a package maintainers `ninja` over the `pip` one is necessarily a good idea?

Reply:
Ninja isn't written in python, it is mostly C++, which would explain why I was confused that you could get it from pip at all.

The python package bundles a 1.7.2 executable (latest is 1.8.2) and some utils for generating .ninja files.

If we want the latest binary then we'd be looking at getting it from https://ninja-build.org/